src/scripts/components/DropdownWrapper.jsx

```diff
@@ -34,8 +34,8 @@ class DropdownWrapper extends React.Component {
       onComplete: () => {
         const $input = $dropdown.querySelector('input[type="text"]');
 
-        // If dropdown contains input (for filtering), focus on that
-        if ($input) {
+        // If dropdown contains input (for filtering), focus on that if autoFocusInput props is true or undefined
+        if ($input && (this.props.autoFocusInput === undefined || this.props.autoFocusInput !== false)) {
```

Comment:
Use React's `defaultProps` to default `autoFocusInput` value to `true`.

Reply:
@alexgsummer Updated as suggested. Thanks this was useful :+1: